src/lib/rest/rest.cpp

```diff
@@ -1410,7 +1410,22 @@ static int connectionTreat
   }
   else
   {
-    serveFunction(ciP);
+    if (metricsMgr.isOn())
+    {
+      struct timeval  start;
+      struct timeval  end;
+
+      gettimeofday(&start, NULL);
```

Comment:
The end time moved to the end of the function requestCompleted, which is the last point possible for the broker. Only MHD knows more.

Fixed in b95465e